### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by amitch6097** [amitch6097/MI-449-SS17-741-html-semantic-elements-5BSgA-#1]
@egillespie Can you take a look at this? 
I am not sure if I was suppose to style this page with CSS.  If so, just let me know and I can do that!
Thanks!
It's [hosted here](https://amitch6097.github.io/MI-449-SS17-741-html-semantic-elements-5BSgA-/) and meets the following criteria:

- [x] Place a `nav` element at the top of the page that includes (fake) links to other pages
- [x] Use the `main` element appropriately
- [x] Include three products your store sells, all listed on the same Products page
- [x] For each product, correctly use an `article`, `header`, and `footer` element
- [x] Use `figure` and `figcaption` to show what each product looks like
- [x] Each product includes a `nav` element with one or more links to related products (the links don't have to go anywhere)
- [x] Use an `aside` in one or more products to advertise a promotion or discount for that product

**Comment by egillespie** [comment]
Nope, there was no need to style anything for this project! Your project is also structured really well, I don't have any improvements to recommend regarding the semantic tags!

In your top-most `nav`, I would recommend you put the links inside a list with `li` tags. You could then style this lists however you want, without changing the HTML further. There are styles that can even add the pipes between each anchor so the result looks identical to what you have now, but using CSS instead of HTML.

If you're curious to know how, reach out to me in Slack!

That's not a requirement for this project, though, so let's ship it. :shipit:
